### PR TITLE
Comment removal

### PR DESCRIPTION
**Pull request by amdshehab**
removes comments
can only remove your own comments

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -27,30 +26,32 @@
...
-        <div class='comment-col-left float-left'>
-          <%= form_for([post, post.comments.build]) do |comment| %>
-            <%= comment.hidden_field(:user_id, value: current_user.id) %>
-            <%= comment.text_field :body, placeholder: 'Write a comment... ✏️', class: 'comment-input-box float-left' %>
-            <%= comment.submit class: 'hidden' %>
-          <% end %>
-        </div>
-      </div>
-
+      <% end %>
```
Comment (adamjohnsnow) [inline app/views/posts/index.html.erb]: Indentation?

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -27,30 +26,32 @@
...
+           <% end %>
+         </div>
+       </div>
+     </div>
+   </div>
+ <% end %>
+
+   <%= link_to new_post_path, id: "new-posts" do %>
+     New post
+   <% end %>
```
Comment (adamjohnsnow) [inline app/views/posts/index.html.erb]: More indentation inconsistency

**Review by adamjohnsnow — CHANGES_REQUESTED** [review]
Code looks good but some indentation inconsistencies have crept in.

**Review by adamjohnsnow — APPROVED** [review]
I think that looks better, @neoeno may have other thoughts...

File: spec/features/user_can_comment_on_posts_spec.rb [hunk 3]
```diff
@@ -28,4 +28,31 @@
...
+      sign_in
+      visit '/posts'
+      click_link 'New post'
+      fill_in 'Message', with: 'First Post'
+      click_button 'Submit'
+      fill_in 'comment[body]', with: 'This is a comment, look at my comment'
+      click_button 'Create Comment'
+      click_link 'Sign out'
+      sign_up_with('ahmed', 'egypt@egypt.com', 'tron123')
+      expect(page).to_not have_content('Delete Comment')
```
Comment (neoeno) [inline spec/features/user_can_comment_on_posts_spec.rb:55]: Nice test!

File: app/assets/stylesheets/comments.css [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (neoeno) [inline app/assets/stylesheets/comments.css]: Not needed

**Review by neoeno — CHANGES_REQUESTED** [review]
Indentation isn't good here. I'll be merging a branch shortly that will help you out on this.

Other than that — looks pretty good! I really like the attention to detail with the 'can't delete own comments' test!

**Review by neoeno — APPROVED** [review]

**Comment by neoeno** [comment]
Nice work! 🚢

**Pull request merged**